docker/Ansible/ansible_container.yaml

```diff
@@ -0,0 +1,122 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://cloudify.co/spec/cloudify/5.0.5/types.yaml
+  - plugin:cloudify-docker-plugin
+
+inputs:
+
+  docker_host:
+    type: string
+    default: ''
```

Comment:
Should we provide a default here? Maybe we should require the user to provide this IP?

Reply:
removed the default , and made the required